src/options.ts

```diff
@@ -99,3 +106,29 @@ export interface CSharpOutputOptions {
    */
   namespace: string;
 }
+
+export interface GoLangOutputOptions {
+  /**
+   * Base root directory.
```

Comment:
This is not accurate in case users want output to go to a nested directory

Reply:
Added some more information here aswell ✅